components/brave_extension/extension/brave_extension/background/today/urls.ts

```diff
@@ -19,14 +21,35 @@ function getHostname (): Promise<string> {
   })
 }
 
+function getRegionURLPart (): Promise<string> {
```

Comment:
I think the name might be misleading - this is basically an optional subdomain, right? Sorry to nitpick, but would it be better as `getRegionSubdomain` (with empty being `en` basically)

Reply:
If it's not a subdomain, definitely ignore this (and apologies!). I see it referenced below in the source list